includes/class-wc-payments-checkout.php

```diff
@@ -211,18 +211,18 @@ public function get_payment_fields_js_config() {
 		 */
 		$payment_fields = apply_filters( 'wcpay_payment_fields_js_config', $js_config );
 
-		$payment_fields['accountDescriptor']        = $this->gateway->get_account_statement_descriptor();
-		$payment_fields['addPaymentReturnURL']      = wc_get_account_endpoint_url( 'payment-methods' );
-		$payment_fields['gatewayId']                = WC_Payment_Gateway_WCPay::GATEWAY_ID;
-		$payment_fields['isCheckout']               = is_checkout();
-		$payment_fields['paymentMethodsConfig']     = $this->get_enabled_payment_method_config();
-		$payment_fields['testMode']                 = WC_Payments::mode()->is_test();
-		$payment_fields['upeAppearance']            = get_transient( WC_Payment_Gateway_WCPay::UPE_APPEARANCE_TRANSIENT );
-		$payment_fields['wcBlocksUPEAppearance']    = get_transient( WC_Payment_Gateway_WCPay::WC_BLOCKS_UPE_APPEARANCE_TRANSIENT );
-		$payment_fields['cartContainsSubscription'] = $this->gateway->is_subscription_item_in_cart();
-		$payment_fields['currency']                 = get_woocommerce_currency();
-		$cart_total                                 = ( WC()->cart ? WC()->cart->get_total( '' ) : 0 );
-		$payment_fields['cartTotal']                = WC_Payments_Utils::prepare_amount( $cart_total, get_woocommerce_currency() );
+		$payment_fields['accountDescriptor']          = $this->gateway->get_account_statement_descriptor();
+		$payment_fields['addPaymentReturnURL']        = wc_get_account_endpoint_url( 'payment-methods' );
+		$payment_fields['gatewayId']                  = WC_Payment_Gateway_WCPay::GATEWAY_ID;
+		$payment_fields['isCheckout']                 = is_checkout();
+		$payment_fields['paymentMethodsConfig']       = $this->get_enabled_payment_method_config();
+		$payment_fields['testMode']                   = WC_Payments::mode()->is_test();
+		$payment_fields['upeAppearance']              = get_transient( WC_Payment_Gateway_WCPay::UPE_APPEARANCE_TRANSIENT );
+		$payment_fields['wcBlocksUPEAppearance']      = get_transient( WC_Payment_Gateway_WCPay::WC_BLOCKS_UPE_APPEARANCE_TRANSIENT );
+		$payment_fields['cartContainsSubscription']   = $this->gateway->is_subscription_item_in_cart();
+		$payment_fields['currency']                   = get_woocommerce_currency();
+		$cart_total                                   = ( WC()->cart ? WC()->cart->get_total( '' ) : 0 );
+		$payment_fields['cartTotal']                  = WC_Payments_Utils::prepare_amount( $cart_total, get_woocommerce_currency() );
```

Comment:
Check these whitespace changes. I don't _think_ they're needed.

Reply:
These should be fixed now. For some reason, the PHP linter isn't flashing on my local editor. 😠 Ma b.